source/chapters/microphones.rst

```diff
@@ -411,3 +411,27 @@ input.
 
 DJ controllers typically do not supply phantom power required for condenser
 microphones. Stand-alone audio interfaces typically do supply phantom power.
+
+Microphone Ducking
+------------------
+While using Mixxx, you might want to talk through your microphone as the music plays, therefore, you might need the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this.
+
+.. figure:: ../_static/microphone_ducking.png
+   :align: center
+   :width: 40%
+   :figwidth: 90%
+   :alt: The microphone ducking section
+   :figclass: pretty-figures
+
+   The microphone ducking controls
+
+The Strength knob adjusts the microphone talkover ducking strength and its behaviour depends on any of the Microphone talkover modes described below.
+Clicking  on the :guilabel:`Auto` button will switch it to any of three modes:
+
+   * Off: When you talk through your microphone, the music  volume does not reduce.
+   * Auto: Talking through your microphone will automatically reduce the music volume. You can adjust the amount to which the volume is automatically reduced with the Strength knob.
+   * Manual: This sets a fixed level to which the music volume reduces to, regardless of the volume of the microphone input. When you talk through your microphone, you can manually set the amount to which the music volume is reduced by turning the Strength knob to a level that suits you.
```

Comment:
I forget if the knob represents the volume is reduced *to*, or the amount the volume is reduced *by*.  I'm also not sure if those are identical things :).

Reply:
"Manual: This sets a fixed level to which the music volume reduces to when the Talk button is on, regardless of the volume of the microphone input. When you talk through your microphone, you can manually set the amount to which the music volume is reduced by turning the Strength knob to a level that suits you."

I think that correctly describes how ducking works?  I honestly don't remember

Reply:
I have edited Manual mode to differentiate it from the description. The new description teaches the user how to adjust the music volume when he talks, using the Strength knob in manual mode.